src/client/pages/[locale]/installer/02.tsx

```diff
@@ -192,6 +192,10 @@ export default function Page(_properties: InferGetStaticPropsType<typeof getStat
 									url: "https://blibla-captain-assets.s3.eu-central-1.amazonaws.com/portable-git-win.7z",
 									destination: "portable-git",
 								},
+								{
```

Comment:
How large is this file? Let's try to sort the files by size as this will slightly improve time used in unpacking.

Maybe we can add comments on expected file sizes like `~50MB` or `>4GB`.

Reply:
The files are already sorted by size (huge > small). I added the `size`, so we also see this.